source/import-export.txt

```diff
@@ -6,6 +6,14 @@ Import and Export Data
 
 .. default-domain:: mongodb
 
+.. facet::
+   :type: programming_language
+   :values: json
```

Comment:
[question] Is JSON a programming language?

Reply:
JSON falls within the schema.org definition for programmingLanguage, so it's listed as one of our language values on the [Official Taxonomy](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy?src=contextnavpagetreemode).